snippets/product-card.liquid

```diff
@@ -1,21 +1,53 @@
 {% style %}
   .yc-product-card {
```

Comment:
please include these style in separate css file since you don't need any dynamic values